muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/workspace/WorkspaceJsPluginMessages_en.properties

```diff
@@ -268,17 +268,36 @@ plugin.workspace.journal.deleteEntry.dialog.cancelButton = Cancel
 plugin.workspace.journal.deleteEntry.dialog.description = Are you sure you want to delete journal entry?
 plugin.workspace.journal.newEntry.title = New Journal Entry
 plugin.workspace.journal.editEntry.title = Edit Journal Entry
+plugin.workspace.journal.comments.title = Comments
```

Comment:
Same as above. Possible discrepancies against `WorkspaceJsPluginMessages_fi.properties`

Reply:
Should be fixed now